src/main/java/fi/vrk/xroad/monitor/actor/ResultCollectorActor.java

```diff
@@ -58,8 +58,7 @@ public Receive createReceive() {
   }
```

Comment:
Actually not about this specific line, but the whole class....isnt the name "ResultCollectorActor" kind of ambiguous? MonitorDataActor also "collects" some kind of "results"? Maybe mention SecurityServerInfo or something more specific in the name to make the purpose more clear?

Reply:
Also name of ResultCollectorActor.MonitorDataResult is confusing to me, makes you think it is something MonitorDataActor produces?